### ISSUE TITLE
need support for iMac19.2 SMBIOS

### ISSUE THREAD

**Issue by cattyhouse** [RehabMan/OS-X-USB-Inject-All#21]
Hi, I found that the SSDT-UIAC i made for my Motherboard does not work properly on iMac 19,2 SMBIOS. 

I have HS 01 02 03 05 06 07 08 09 14 and SS 01 02 03 04 06 defined on iMac18,3 macOS 10.14.4,  recently i upgraded to 10.14.5 beta1 and use SMBIOS iMac 19,2. the ports in iojones was displayed as:
HS 01 02 03 04 05 06 07 08 09 10 11 14 SS 01 02 03, thus SS 04 05 06 does not work.

**Comment by RehabMan** [comment]
TODO: Add latest models to the Info.plist script.

**Comment by Appyx** [comment]
I had the same issue.

I fixed it by cloning the project into Xcode, and then I added the following to the `generate_Info_plist.sh `file:

`mergeModelData "iMac19,2"`

Additionally, I had to change the sdkroot of both build targets to macosx (from macosx10.11):

PreBuild/Build Settings/SDKROOT/macosx
USBInjectAll/BuildSettings/Base SDK -> macOS

After these three steps I built the targets and copied the kext to my EFI folder.

I hope this helps until @RehabMan releases the new version.

Edit: I also attached the compiled version, which should work OOB.
[USBInjectAll.kext.zip](https://github.com/RehabMan/OS-X-USB-Inject-All/files/3267860/USBInjectAll.kext.zip)
